uber/templates/preregistration/preregbase.html

```diff
@@ -7,7 +7,7 @@
 <link href="../static/theme/prereg.css" rel="stylesheet">
 <script type="text/javascript">
     var checkCap = function () {
-        $.getJSON('check_prereg', function(check) {
+        $.getJSON('../preregistration/check_prereg', function(check) {
```

Comment:
Since we're not extending the preregbase template from an area outside the prereg site section, we need to be more explicit in this URL so it will work in both places.